src/CLIArchiveCommand.ts

```diff
@@ -23,7 +23,7 @@ export default class CLIArchiveCommand implements ArchiveCommand {
   }
 
   async generateOpenAPI(directory: string): Promise<void> {
```

Comment:
OK yeah, so this directory is actually supposed to be `appmap_dir`. Good catch :-)

Two other things occur to me when I see this:

1) Why is it bothering at all? The openapi command should be able to find `appmap_dir` on its own. I guess it's making sure to honor the appmap dir from `appmap_archive.json`.

2) The `archive` command actually [generates OpenAPI](https://github.com/getappmap/appmap-js/blob/360eb81271255cf59139f438db5f896e9ca7c99b/packages/cli/src/cmds/archive/analyze.ts#L56) itself - so I think this code in the `archive-action` isn't actually needed.

Reply:
Yea on point #1 i was thinking something similar that it would likely be in the appmap.yml? but not entirely sure.  And then yea on point #2 if that happens in the archive then this would be duplicate - i'm nearly sure in my GH action test runs the archive runs after merge anyway.

Reply:
Also - looking at this code https://github.com/getappmap/archive-action/blob/main/src/merge.ts#L93-L97 - The OpenAPI is definitely being generated on the `Save Archive` command per [this run](https://github.com/getappmap/Misago/actions/runs/6053445373/job/16428953910#step:8:32).

If we merged those OpenAPIs that would also avoid the issue i see right now in that project where generating an OpenAPI on the merged appmaps causes an OOM